### PR TITLE
EVA-3227  Add file size to metadata Json

### PR DESCRIPTION
**Pull request by nitin-ebi**
The docker validator test is failing as I have not published the new docker image with the updated validation.nf file. 
(I have tested in the local env with the new image and it passes)
I thought of doing it after first review, in case there are any review comments on the validation.nf file.

### CODE REVIEW

File: eva_sub_cli/validators/validator.py [hunk 1]
```diff
@@ -514,17 +514,22 @@ def _convert_metadata_attribute(self, sheet, json_attribute, xls2json_conf):
                 return attribute
 
     def _collect_md5sum_to_metadata(self):
```
Comment (apriltuesday) [inline eva_sub_cli/validators/validator.py:516]:
```suggestion
    def _collect_file_info_to_metadata(self):
```

**Review by apriltuesday — APPROVED** [review]

File: eva_sub_cli/validators/validator.py (hunk 1 above)
Comment (nitin-ebi) [reply eva_sub_cli/validators/validator.py]: updated

File: eva_sub_cli/validators/docker_validator.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 logger = logging_config.get_logger(__name__)
 
 container_image = 'ebivariation/eva-sub-cli'
-container_tag = 'v0.0.1.dev10'
+container_tag = 'v0.0.1.dev11'
```
Comment (tcezard) [inline eva_sub_cli/validators/docker_validator.py:15]: `dev11` does not exist in dockerhub which is why the test are failing

Comment (nitin-ebi) [reply eva_sub_cli/validators/docker_validator.py:15]: published the new version in dockerhub

**Review by tcezard — APPROVED** [review]

**Pull request merged**